scss/styles.scss

```diff
@@ -102,3 +102,42 @@
   align-self: center;
   width: 200px;
 }
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  padding: 10px;
+  width: 100%;
+}
+
+.footer-title {
+  color: $white;
+  font-size: $font-big;
+  margin-bottom: 25px;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 300px;
+  width: 100%;
+}
+
+.footer-input {
+  border-radius: 5px;
+  height: 3vh;
```

Comment:
vh represents a percentage of the screen height, you should only use `vh` when using known values like `50vh` or  `100vh`, this is because it makes the element vary a lot depending on the size of the screen. for things like this, a pixel height is a good choice, I think something like 30 or 40 px will fit